### PR TITLE
feat: dummy sha256 plugin using hipcheck-sdk

### LINKED ISSUE #423: Transition dummy plugins to use the SDK

### PR DESCRIPTION
**Pull request by patrickjcasey**
This branch implements the `dummy/sha256` plugin with the `hipcheck-sdk`

This will close #423

### CODE REVIEW

**Comment by j-lanson** [comment]
@mchernicoff can you take a look?

File: plugins/dummy_sha256_sdk/src/main.rs [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+		Ok(())
+	}
+
+	fn default_policy_expr(&self) -> hipcheck_sdk::prelude::Result<String> {
+		Ok("".to_owned())
+	}
+
+	fn default_query(&self) -> Option<DynQuery> {
+		Some(Box::new(Self))
+	}
```
Comment (j-lanson) [inline plugins/dummy_sha256_sdk/src/main.rs:68]: I don't think you're supposed to implement this function yourself. The default query will be the `NamedQuery` with `name.empty()`. This plugin won't have a default query, unless you want to change `NamedQuery` below and update `dummy_rand_data_sdk` to just query `dummy/sha256`.

Comment (patrickjcasey) [reply plugins/dummy_sha256_sdk/src/main.rs]: I think it's cleaner to just not implement this function

**Review by j-lanson — APPROVED** [review]
Looks good! Thanks Patrick

**Pull request merged**